assignments/arrays.js

```diff
@@ -63,34 +63,68 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+//console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+console.log(inventory[32]);
```

Comment:
You did log out the year/make/model, but not in the correct format. But based on your work on Challenge 2, I think you could complete this one using a string using a [template literal.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)